src/metadata_tool/tools/reports.clj

```diff
@@ -169,7 +169,7 @@
                                                                            (md/activities-metadata)))
         ]
     (doall (map #(send-email-to-pmc (:program-id %)
-                                    (str (:program-short-name %) " PMC Report as at " now-str)
+                                    (str (:program-name %) " PMC Report for March " now-str)
```

Comment:
Hardcoding the month name (```March```) seems like a mistake.  Why not let [```clj-time/format``` string](https://github.com/clj-time/clj-time#clj-timeformat) do all the hard work?

i.e. something like:
```clojure
  (let [now-str (tf/unparse (tf/formatter "MMMM yyyy") (tm/now))
  ...
  (str (:program-name %) " PMC Report for " now-str)
```

Reply:
yes I do agree. I thought of doing it parametrical instead of a text but it was too late in the day of making this change. Will make it tomorrow if Mao doesn't do it before me